clicache/src/Cache.cpp

```diff
@@ -57,6 +56,21 @@ namespace Apache
         m_typeRegistry = gcnew Apache::Geode::Client::TypeRegistry(this);
       }
 
+      Cache::~Cache() { //Destructor - deterministic
+        if (!_disposed) {
+          //Clean-up managed resources
+          this->!Cache();
+          _disposed = true;
```

Comment:
Member variables should be consistently formatted. 

Should be `m_disposed` to be consistent with old convention or `disposed_` for current convention.

Reply:
Fixed.